src/hearings/models/refData.model.ts

```diff
@@ -6,6 +6,7 @@ export interface RefDataModel {
   hintTextCY: string;
   order: number;
   parentKey: string;
+  selected?: boolean;
```

Comment:
this is a generic model from refdata API, what is the purpose of adding this selected?

Reply:
We need to know which are selected if this model is used as checkboxes. So, added it as optional